editor/import/resource_importer_imagefont.h

```diff
@@ -53,6 +53,9 @@ class ResourceImporterImageFont : public ResourceImporter {
 	virtual Error import(const String &p_source_file, const String &p_save_path, const HashMap<StringName, Variant> &p_options, List<String> *r_platform_variants, List<String> *r_gen_files = nullptr, Variant *r_metadata = nullptr) override;
 
 	ResourceImporterImageFont();
+
+private:
+	String CorrectOutlierSplitters(const String &range);
```

Comment:
```suggestion
	String correct_outlier_splitters(const String &p_range);
```
Prefix arguments with `p_` and use `snake_case`